ontology-engine/graph-core_2.12/src/test/scala/org/sunbird/graph/external/ExternalPropsManagerTest.scala

```diff
@@ -49,17 +49,22 @@ class ExternalPropsManagerTest extends BaseSpec {
 
     "savePropsWithTtl" should "create a cassandra record with ttl successfully" in {
         val request = new Request()
-        request.setObjectType("Lock")
+        request.setObjectType("ObjectCategoryDefinition")
```

Comment:
Why we change the Object Type?

Reply:
@maheshkumargangula The test case was failing, so I referred to the 'saveProps' test case and added the same inputs to this one